en/plugins.rst

```diff
@@ -94,11 +94,15 @@ Additionally, you will need to tell Composer to refresh its autoloading cache:
 If you are unable to use Composer for any reason, you can also configure
 autoloading with ``Plugin``::
 
-    // In App\Application::bootstrap() add.
-    $this->addPlugin('ContactManager', ['autoload' => true]);
+    Plugin::load('ContactManager', ['autoload' => true]);
 
-    // Prior to 3.6.0 you need to use Plugin::load()
 
+.. deprecated:: 3.7.0
+    Plugin::load() and `autoload` option  are deprecated.
```

Comment:
do we need double ` here?

Reply:
I dont know enough about formatting on docs :(

Reply:
See the above text :)

Reply:
I did not  run it in local

Reply:
no need to.

Reply:
this is within a box (deprecated box) I'm not sure really about affect it on ``  :)
if you are sure I can change it